website/docs/docs/faq.md

```diff
@@ -59,7 +59,26 @@ sidebar_position: 100
    resource dependency management and customization, paving the way for a simple
    and scalable way to create complex custom resources for Kubernetes.
 
-5. **Can I use this in production?**
+5. **How do I use KRO resources with ArgoCD?**
+
+   To use KRO resources with ArgoCD, you need to add a specific tracking annotation
+   to all templated resources in your ResourceGraphDefinition. 
+   
+   The following code needs to be added to each templated resource:
+
+   ```yaml
+   annotations:
+     argocd.argoproj.io/tracking-id: ${schema.metadata.?annotations["argocd.argoproj.io/tracking-id"]}
+   ```
+
+   This annotation allows ArgoCD to properly track and manage the resources
+   created by KRO instances. 
+   
+   Note that this approach only works when using ArgoCD's default resource tracking 
```

Comment:
I think this works with `annotation` and `annotation+label`. Even `app.kubernetes.io/instance` should work if you set this label right? Would be cool if you could confirm this!

Reply:
It kind a does... With `annotation+label` the setup is that label is used more for infotmative purposes to different apps

> label is not used for tracking purposes, and the value is still truncated if longer than 63 characters. The annotation argocd.argoproj.io/tracking-id is used instead to track application resources

For just `label` its well documented `Argo CD uses the app.kubernetes.io/instance label` 

I have expanded the text to highlight the choice and re-point people to the documentation